GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -171,6 +174,22 @@ private void ReloadCommitInfo()
 
             if (AppSettings.CommitInfoShowContainedInTags)
                 ThreadPool.QueueUserWorkItem(_ => loadTagInfo(_revision.Guid));
+
+
+            commitSignPicture.Visible = tagSignPicture.Visible = AppSettings.ShowGpgInformation;
+
+            if (AppSettings.ShowGpgInformation)
+            {
+                ThreadPool.QueueUserWorkItem(_ => loadCommitSignInfo(_revision));
```

Comment:
the codebase has switched to .NET 4.6.1, please use async/await rather than thread pool.
rebase on top of master if necessary.